packages/ui/core/Button/Button.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {DefaultTheme, withTheme} from 'styled-components/native';
+
+import {Touchable, StyledLinearGradient, Text} from './styles';
+import {
+  ButtonComponentType,
+  ButtonActive,
+  ButtonDisabled,
+  ButtonTextColor,
+  ButtonProps,
+  gradientConfig,
+  SMALL as TEXT_SMALL,
+} from './types';
+
+function getColor(
+  theme: DefaultTheme,
+  disabled?: ButtonDisabled,
+  color?: ButtonTextColor,
+  active?: ButtonActive,
+): string {
+  if (disabled) return theme.grey;
+  if (color) return color;
+  if (active) return theme.white;
+
+  return theme.greenLight;
+}
+
+const Button: ButtonComponentType = ({text, theme, ...props}) => {
+  const {active, gradient, disabled, rounded, outline, size, textColor: color} = props;
+
+  const textColor = getColor(theme, disabled, color, active || gradient);
+  const textSize = size === TEXT_SMALL ? 'body' : 'title-3';
+  const {locations, start, end} = gradientConfig;
+
+  if (disabled && (active || gradient)) console.warn("You can't use `gradient` prop when your button is disabled.");
```

Comment:
Usually is good to show these kinds of 'warn for developers' only when is on development mode

Reply:
Exactly. Thanks! :D